### PR TITLE
:sparkles: Add forgot password page

### PR DESCRIPTION
**Pull request by infoucarsinan**
Add forgot password page

### CODE REVIEW

File: src/app/pages/ForgotPassword/ForgotPassword.module.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  color: var(--alert-color);
+  text-align: center;
+}
+
+.forgotPasswordForm__ForgotPasswordTitle {
+  text-transform: uppercase;
+  font-weight: 500;
+}
+
+.forgotPasswordForm_RegisterLink {
```
Comment (MaxMuecket) [inline src/app/pages/ForgotPassword/ForgotPassword.module.css:30]:
```suggestion
.forgotPasswordForm__RegisterLink {
```

File: src/app/pages/ForgotPassword/ForgotPassword.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import ForgotPassword from "./ForgotPassword";
+import { Meta } from "@storybook/react";
+
+export default {
+  title: "Page/ForgotPassword",
+  parameters: {
+    layout: "fullscreen",
+  },
+} as Meta;
```
Comment (MaxMuecket) [inline src/app/pages/ForgotPassword/ForgotPassword.stories.tsx:10]: What does "as Meta" do?

**Review by MaxMuecket — APPROVED** [review]
Good job! ✌️

File: src/app/pages/ForgotPassword/ForgotPassword.module.css (hunk 1 above)
Comment (infoucarsinan) [reply src/app/pages/ForgotPassword/ForgotPassword.module.css:30]: Thanks :-)

File: src/app/pages/ForgotPassword/ForgotPassword.stories.tsx (hunk 2 above)
Comment (infoucarsinan) [reply src/app/pages/ForgotPassword/ForgotPassword.stories.tsx:10]: Come from Leon!

File: src/app/pages/ForgotPassword/ForgotPassword.module.css (hunk 1 above)
Comment (infoucarsinan) [reply src/app/pages/ForgotPassword/ForgotPassword.module.css:30]: ok

**Pull request merged**